### PR TITLE
Fix [BUG] #1750

### PR DESCRIPTION
**Pull request by Ben-Epstein**
## What changes are proposed in this pull request?
 
This is a PR to fix issue #1750 adding the JRE installation to the Azureml docker image. This shouldn't have any breaking changes and testing locally has been shown to fix the issue.
@abaveja313 
 
## How is this patch tested?
 
(Details)
 
### Is this a user-facing change? 

- [X] No. You can skip the rest of this section.
- [ ] Yes. Give a description of this change to be included in the release notes for MLflow users.
 
(Details in 1-2 sentences. You can just refer to another PR with a description if this PR is part of a larger change.)
 
### What component(s) does this PR affect?
 
- [ ] UI
- [ ] CLI 
- [ ] API 
- [ ] REST-API 
- [ ] Examples 
- [ ] Docs
- [ ] Tracking
- [ ] Projects 
- [ ] Artifacts 
- [X] Models 
- [ ] Scoring 
- [X] Serving
- [ ] R
- [ ] Java
- [X] Python

### How should the PR be classified in the release notes? Choose one:
 
- [ ] `rn/breaking-change` - The PR will be mentioned in the "Breaking Changes" section
- [X] `rn/none` - No description will be included. The PR will be mentioned only by the PR number in the "Small Bugfixes and Documentation Updates" section
- [ ] `rn/feature` - A new user-facing feature worth mentioning in the release notes
- [ ] `rn/bug-fix` - A user-facing bug fix worth mentioning in the release notes
- [ ] `rn/documentation` - A user-facing documentation change worth mentioning in the release notes

### CODE REVIEW

**Comment by abaveja313** [comment]
Looks good!

File: mlflow/azureml/__init__.py [hunk 1]
```diff
@@ -248,7 +248,8 @@ def _create_dockerfile(output_path, mlflow_path=None):
                         Dockerfile command for MLflow installation will install MLflow from this
                         directory. Otherwise, it will install MLflow from pip.
     """
-    docker_cmds = ["RUN pip install azureml-sdk"]
+    docker_cmds = ["RUN apt-get update && apt-get install default-jre"]
```
Comment (dbczumar) [inline mlflow/azureml/__init__.py]:
```suggestion
    docker_cmds = ["RUN apt-get update && apt-get install -y default-jre"]
```

Comment (dbczumar) [reply mlflow/azureml/__init__.py]: Without the `-y` flag, installation aborts due to a lack of affirmative installation input:
```
Do you want to continue? [Y/n] Abort.
The command '/bin/sh -c apt-get update && apt-get install default-jre' returned a non-zero code: 1
2019/08/22 19:54:46 Container failed during run: acb_step_0. No retries remaining.
failed to run step ID: acb_step_0: exit status 1
```

**Review by dbczumar — APPROVED** [review]
@Ben-Epstein This looks good to me after the inline suggestion is committed. I tested this change by training a Spark model, building an image, and deploying it to ACI.

**Comment by Ben-Epstein** [comment]
@dbczumar fixed, thanks for review.

**Pull request merged**